src/t8_vec.h

```diff
@@ -186,6 +186,20 @@ t8_vec_diff (const double vec_x[3], const double vec_y[3], double diff[3])
   }
 }
 
+/**
+ * Check the equality of two vectors with respect to the euclidean norm. 
+ * 
+ * \param[in] vec_x 
+ * \param[in] vec_y 
+ * \param[in] precision 
+ * \return true, if the vectors are equal up to \a precision 
+ */
+static inline int
+t8_vec_eq (const double vec_x[3], const double vec_y[3], const double precision)
+{
+  return t8_vec_dist (vec_x, vec_y) < precision;
+}
+
```

Comment:
Why are you not checking |x[i] - y[i]| < eps?
Seems like less computational effort...

Reply:
Changed it to an elementwise check.